authors/settings.py

```diff
@@ -169,6 +170,8 @@
 
 STATIC_URL = '/static/'
 
+CORS_ORIGIN_ALLOW_ALL = True
```

Comment:
Should this be true?

Reply:
We decided to whitelist all requests during development. This will allow us to work with different local server configurations and still query the API